.babelrc

```diff
@@ -16,6 +20,9 @@
     "development": {
       "plugins": ["react-hot-loader/babel"]
     },
+    "production": {
+      "plugins": [["lodash", { "id": ["lodash"] }]]
+    },
     "test": {
       "presets": ["@babel/preset-env", "@babel/preset-react"],
```

Comment:
do we need this? we have both presets in the global config as well, this looks like it just resets preset-env to the default for `test`. do you know why we have it?

Reply:
Likely it came to the project through the initial work and we forgot to remove `@babel/preset-env` from here. Let's do that now. 👍

Reply:
It's the same with React preset as well.

Reply:
Ok, changed.